unlock-app/src/components/creator/lock/KeyList.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import styled from 'styled-components'
+import UnlockPropTypes from '../../../propTypes'
+import { expirationAsDate } from '../../../utils/durations'
+
+// TODO add pagination
+export function KeyList ({ keys }) {
+  return (
+    <KeyTable>
```

Comment:
You and the prefixes ;) I think there is no need to prefix everything with `Key` here because all of this is only local to that file ;p